src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,114 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import java.util.ArrayList;
+
+
+import util.DukeException;
+
+public class Deadline extends Task {
+
```

Comment:
I like the implementation here, but it would be better if we have a Javadoc for the class as well as its methods. :D